### PR TITLE
Added tooltips for match indices

### PR DESCRIPTION
**Pull request by lemongod**

### CODE REVIEW

File: views/match/match_index.jade [hunk 1]
```diff
@@ -22,20 +22,20 @@ block match_content
...
-              th LH
-              th DN
-              th GOLD
-              th GPM
-              th XPM
-              th HD
-              th TD
-              th HH
-              th Items
+              th: abbr(title="Valve's match making rating") Rating
```
Comment (howardc93) [inline views/match/match_index.jade]: I would just call this "Solo MMR"

File: views/match/match_index.jade [hunk 2]
```diff
@@ -22,20 +22,20 @@ block match_content
...
+              th: abbr(title="Level of hero") LV
+              th: abbr(title="Number of kills by hero") K
+              th: abbr(title="Number of deaths of hero") D
+              th: abbr(title="Number of assists by hero") A
+              th: abbr(title="Number of last hits by hero") LHs
+              th: abbr(title="Number of denied enemy creeps by hero") DN
```
Comment (howardc93) [inline views/match/match_index.jade]: "denied creeps".  You don't deny enemy creeps

File: views/match/match_index.jade [hunk 3]
```diff
@@ -22,20 +22,20 @@ block match_content
...
+              th: abbr(title="Amount of gold amassed") GOLD
```
Comment (howardc93) [inline views/match/match_index.jade]: amassed?  I'm okay with it but maybe simpler just to say "Total gold farmed"

File: views/match/match_index.jade [hunk 4]
```diff
@@ -22,20 +22,20 @@ block match_content
...
+              th: abbr(title="Gold received by hero per minute") GPM
+              th: abbr(title="Experience gained by hero per minute") XPM
+              th: abbr(title="Amount of damage dealt to enemy heroes") HD
+              th: abbr(title="Amount of tower damage dealt") TD
+              th: abbr(title="Amount of health healed") HH
```
Comment (howardc93) [inline views/match/match_index.jade]: This is hero healing, so it includes allies.  may want to clarify this--"Amount of health restored to heroes"

**Comment by howardc93** [comment]
thanks for this @lemongod .

@albertcui should review and merge.

File: views/match/match_index.jade [hunk 5]
```diff
@@ -22,20 +22,20 @@ block match_content
...
-              th HD
-              th TD
-              th HH
-              th Items
+              th: abbr(title="Solo MMR") Rating
+              th: abbr(title="Level of hero") LV
+              th: abbr(title="Number of kills by hero") K
+              th: abbr(title="Number of deaths of hero") D
+              th: abbr(title="Number of assists by hero") A
+              th: abbr(title="Number of last hits by hero") LHs
```
Comment (vramana) [inline views/match/match_index.jade]: Why `LHs` ?? I mean we are not writing `Ks` for kills, `DNs` for denies. I think it should stay as `LH`.

Comment (howardc93) [inline views/match/match_index.jade]: I didn't see this.  @lemongod don't change the header abbreviations.

**Comment by howardc93** [comment]
Fixed this for you @lemongod , @albertcui this should be ready to merge.

**Pull request merged**